lib/autoload/send_file.rb

```diff
@@ -0,0 +1,19 @@
+module SendFile
+  # Makes the given file publicly accessible.
+  #
+  # @param [String] file The path to the local file.
+  # @param [String] public_name The file's public name, default set to file's base_name.
+  # @return [String] The url to the public visible file
+  def send_file(file, public_name = File.basename(file))
+    # TODO: Support remote path and publish the file on the load balancing server.
+    downloads_dir = 'downloads/'
+    public_dir = File.join(Rails.public_path, downloads_dir)
+    Dir.mkdir(public_dir) unless File.exist?(public_dir)
```

Comment:
Yah, think I should do that, it also improves the performance, as I don't need to check every time.

 thanks !